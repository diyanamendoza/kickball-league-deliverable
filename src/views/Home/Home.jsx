import React from 'react'

export default function Home() {
    return (
        <div className='home'>
           <h2>Fake Kickball League</h2> 
           <p>Realtalk: I hate kickball.</p>
        </div>
    )
}
