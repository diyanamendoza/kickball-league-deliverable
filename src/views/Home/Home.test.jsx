import { render } from '@testing-library/react'
import Home from './Home'

it('should render a homepage', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
})