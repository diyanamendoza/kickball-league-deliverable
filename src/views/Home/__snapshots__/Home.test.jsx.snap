// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a homepage 1`] = `
<div>
  <div
    class="home"
  >
    <h2>
      Fake Kickball League
    </h2>
    <p>
      Realtalk: I hate kickball.
    </p>
  </div>
</div>
`;
